graylog2-server/src/main/java/org/graylog/security/authzroles/AuthzRolesResource.java

```diff
@@ -88,6 +90,18 @@ public AuthzRolesResource(PaginatedAuthzRolesService authzRolesService) {
         return PaginatedResponse.create("roles", result, query);
     }
 
+
+    @GET
+    @ApiOperation(value = "Get a single role")
+    @Path("/{roleId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @RequiresPermissions(RestPermissions.ROLES_READ)
```

Comment:
this would require the user to have a `roles:read:*` permission.
The specific one underneath is better. I'd just remove this line.
```suggestion
```